test/expect/HasDecompTest.test_has_decomposition.expect

```diff
@@ -1267,6 +1267,8 @@ aten::squeeze_copy.dims
 aten::squeeze_copy.dims_out
 aten::squeeze_copy.out
 aten::sspaddmm.out
+aten::sum.IntList_out
+aten::sum.out
```

Comment:
We can register these on the reference as well

Reply:
Ah, yes that's true. Will update that. But for `sum.out`, I think I will register to the new decomposition since it can be expressed with `sum.IntList_out` as well.